scripts/fetch-config.sh

```diff
@@ -7,4 +7,4 @@ if [[ "$env" != DEV && "$env" != CODE ]]; then
 fi
 
 mkdir -p ~/.configuration-magic/
-aws s3 cp s3://guconf-flexible/editorial-production-metrics/${env}/editorial-production-metrics.conf ~/.configuration-magic/editorial-production-metrics.conf --profile composer --region $region
+aws s3 cp s3://guconf-flexible/editorial-production-metrics/${env}/editorial-production-metrics.conf ~/.gu/editorial-production-metrics.conf --profile composer --region $region
```

Comment:
this is a fix from a previous pr - https://github.com/guardian/editorial-production-metrics/pull/240